UI.py

```diff
@@ -0,0 +1,109 @@
+from getpass import getpass
+# importing 'mysql.connector' as mysql 
+import mysql.connector as mysql
+
+class DbConnectionError(Exception):
+    pass
+
+#DB Connection
+def _connect_to_db():
+    cnx = mysql.connect(
+        host = "localhost",
+        user = "root",
+        passwd = "password",    #Reviewer Password
+        database = "walk2zero"
+    )
+    return cnx
+
+#Function to accept User Input
+def user_reg():
+    first_name = input("Please enter your first name = ")
+    last_name = input("Please enter your last name = ")
+    email = input("Please enter your email address = ")
+    password = getpass("Please enter a desired password = ")
+    new_registration(first_name,last_name,email,password)
+
+#Function to log-in an exisiting user
+def login():
+    check_email = input("Please enter your email address = ")
+    check_password = getpass("Please enter your password = ")
+    verify_login(check_email,check_password)
+
+
+#Function to register new user details into the DB
+def new_registration(first_name,last_name,email,password):
+    #calling DB connection
+    db_connection = _connect_to_db()
+    cur = db_connection.cursor()
+    
+    #defining the Query
+    query = "INSERT INTO users (fname, lname, email, pword) VALUES (%s, %s,%s,%s)"
+    
+    #storing values in a variable
+    values = (first_name, last_name,email,password)
+
+    #executing the query with values
+    cur.execute(query, values)
+    db_connection.commit()  
+    cur.close()
+    print("Successful Registration!")
+    print("")
+
+
```

Comment:
Might be better to have this in a separate file when we get to writing the user interface properly. 

But I really like the exception handling!!!!